ccd-definition/AuthorisationCaseEvent.json

```diff
@@ -167,6 +202,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadStandardDirectionsGatekeeping",
```

Comment:
do we not need to give access to the gatekeeper role here too?

Reply:
same as above